actions/terraform-checks/action.yml

```diff
@@ -10,6 +9,29 @@ runs:
       with:
         terraform_version: ${{ inputs.terraform_version }}
 
+    - name: Cache plugin dir
+      uses: actions/cache@v4
+      with:
+        path: ~/.tflint.d/plugins
+        key: tflint-${{ hashFiles('.tflint.hcl') }}
+
+    - name: "INFO: Setup TFLint"
```

Comment:
Is `INFO:` a yamllint thing?

Reply:
It's more a way to differentiate between logs, maybe not necessary, hangover from wanting to switch between commands with find (CMD+F) for example in a raw file.

Reply:
What other options would we have? I'm thinking if every step is `INFO:` then is it necessary?

Reply:
Good point, just looked at the way this is presented here: https://github.com/oaknational/oak-terraform-actions/actions/runs/10352739526/job/28654004307

<img width="451" alt="image" src="https://github.com/user-attachments/assets/3810ba77-948b-4904-b7bc-2d3b1214323f">

Seems fairly unnecessary given that format.

I'll remove them 👍

Reply:
Removed: 
<img width="386" alt="image" src="https://github.com/user-attachments/assets/388a86d0-259a-429b-969a-6f747d6f32a9">